src/scss/partials/_pin-entry.scss

```diff
@@ -0,0 +1,76 @@
+.pin-login-screen {
+    position: absolute;
+    //top: 100%;
+    width: 100vw;
+    height: 100%;
+    z-index: 101;
+    //transition: top 1s cubic-bezier(.13,.87,.42,.96);
+    //-webkit-transition: top 1s cubic-bezier(.13,.87,.42,.96);
+
+    .pin-input-section, .sending-progress-section {
+
```

Comment:
NIT; white lines between sibblings (eg; `.item-input` and `input` a little down)